treescript/versionmanip.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+"""Treescript version manipulation."""
+
+from distutils.version import StrictVersion
+import logging
+import os
+
+from treescript.exceptions import TaskVerificationError
+from treescript.mercurial import run_hg_command
+from treescript.task import get_version_bump_info
+
+log = logging.getLogger(__name__)
+
+
+ALLOWED_BUMP_FILES = (
+    'browser/config/version.txt',
+    'browser/config/version_display.txt',
+    'config/milestone.txt'
```

Comment:
A few things:
- I don't see a great benefit to specifying the files in the task definition, then constraining them here. Perhaps if we had trees with completely different version file paths, but then we probably want to specify that per tree.
- We bump these files in different ways, and I don't see you allowing for that here. Essentially, `version_display.txt` doesn't match `version.txt` on release trains.
- We may or may not want a check to make sure our version bump follows certain rules. Perhaps we shouldn't allow going backwards; perhaps we shouldn't allow for skipping versions (or skipping more than N versions).